packages/forma-36-react-components/yarn.lock

```diff
@@ -1433,22 +1433,10 @@
     puppeteer "^1.2.0"
     yargs "^11.1.0"
 
-"@contentful/forma-36-fcss@^0.0.31":
```

Comment:
@mohamedturki Please revert the changes in `yarn.lock` - I believe this happens if you've ran `yarn` from the components package directory rather than the root of the monorepo

Reply:
Roger tha'